windows/README.md

```diff
@@ -0,0 +1,77 @@
+# The Flutter Windows Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on the Windows desktop.
+
+This README assumes you have already read [the top level README](../README.md), which contains important information and setup common to all platforms.
+
+## Flutter Location
+
+Contrary to [the top level README](../README.md) Windows cannot be run using a `.flutter_location_config` file and a Flutter tree must be located in the same parent directory as this repo like so:
+
+```
+<parent dir>
+  ├─ flutter (from http://github.com/flutter/flutter)
+  └─ flutter-desktop-embedding (from https://github.com/google/flutter-desktop-embedding)
+```
+
+## Adding Windows as a Target Platform
+
+By default flutter will throw an unknown target platform error if run on Windows. Windows can be added by editing `\path\to\flutter\packages\flutter\lib\src\foundation\platform.dart`.
+
+Change the code around line 37 from this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux) {
+  result = TargetPlatform.android;
+}
+```
+
+to this:
+
+```dart
+if (Platform.isAndroid || Platform.isLinux || Platform.isWindows) {
+  result = TargetPlatform.android;
+}
+```
+
+The flutter app you are trying to run must be rebuilt before it can be run on windows.
```

Comment:
This is true on all platforms, so I'd rather not call it out specifically in the Windows README. The reason it's not in the main README is that the builds on macOS and Linux handle this automatically, modeling what I think is best practice (see comment below).